integration/analytics/Tiltfile

```diff
@@ -0,0 +1,11 @@
+# -*- mode: Python -*-
+
+# If you get push errors, you can change the default_registry.
```

Comment:
last time i wrote an integration test, nick told me that this didn't make sense. so i'll tell you the same thing -- even tho i fully acknowledge that it's copypasta